html/php-pre-challenge2/index.php

```diff
@@ -1,12 +1,39 @@
 <?php
+
 $array = explode(',', $_GET['array']);
 
 // 修正はここから
-for ($i = 0; $i < count($array); $i++) {
+$length = count($array);
+$start = -1;
 
+// don't need to declare $amount as variable as this if statement sets it automatically
+// $amount= count($array); FOLLOWING IF STATEMENT IS AUTOMATIICALLY DECLARED WHEN USING IF STATEMENT
+if( !$amount = $length ){
+    return $array;
```

Comment:
```suggestion
$amount = $length;
```
同じ意味なのでこうしましょう！
functionの中じゃないのにreturnがあるのはおかしいです。
あと `if( !$amount = $length ){` この書き方は読みにくいため。